src/js/chartAuthoring.js

```diff
@@ -188,6 +245,72 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         }]
     });
 
+    floe.chartAuthoring.updateTitleFromEdit = function(pieChart, newTitle) {
+        pieChart.applier.change("pieTitle", newTitle);
+    };
+
+    floe.chartAuthoring.updateDescriptionFromEdit = function(pieChart, newDescription) {
+        pieChart.applier.change("pieDescription", newDescription);
+    };
+
+    // Adds and removes highlight
+    floe.chartAuthoring.highlightPlayingData = function(that) {
```

Comment:
I think the highlighting should happen in pieChart component instead of at the top level. What do you think?